packages/react-dom/src/client/ReactDOMFiberInput.js

```diff
@@ -209,6 +214,16 @@ export function postMountWrapper(element: Element, props: Object) {
   const node = ((element: any): InputWithWrapperState);
 
   if (props.hasOwnProperty('value') || props.hasOwnProperty('defaultValue')) {
+    // Submit or reset inputs should not have their value overridden
+    // with a falsy value, or else this will result in a button with no text,
```

Comment:
This says "falsy" but the code checks for `null` and `undefined`. What should happen for `''`? For `0`? For `false` or `true`? Let's make sure code and comments agree.

Reply:
Good spot. Fixed 😄